### PR TITLE
Another player actions proposal

### PR DESCRIPTION
**Pull request by vermentor**

### CODE REVIEW

File: client/src/GamemasterInGame/PlayerListComponents/PlayerActions.tsx [hunk 1]
```diff
@@ -109,23 +122,52 @@ function RoleChangeMenuItem({
   player: string;
 }) {
   const [, , , setPlayerRole] = useAssignRole();
+  const getPlayerAlignment = useGetPlayerAlignment();
+  const alignment = getPlayerAlignment(player);
+  const role = game.playersToRoles[player];
 
   return (
     <RoleSelect
```
Comment (vermentor) [inline client/src/GamemasterInGame/PlayerListComponents/PlayerActions.tsx:130]: Actually I want header to be simple data and move role select back down into the menu list

**Comment by vermentor** [comment]
Screenshot_20230918-084624_Firefox~2

**Comment by Alphy11** [comment]
Generally like this:
image


I think it should be left corner name, right corner role, beneath statuses.

File: client/src/GamemasterInGame/NotesIcons.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { useDefiniteGame } from "../store/GameContext";
 import { Button, Dialog, Flex, IconButton, Text } from "@radix-ui/themes";
 import { PlayerStatus } from "@hidden-identity/server";
 import { useDeadVote, usePlayerStatuses } from "../store/useStore";
-import { ExtnesionProps } from "../types/radixTypes";
+import { ExtensionProps } from "../types/radixTypes";
```
Comment (Alphy11) [inline client/src/GamemasterInGame/NotesIcons.tsx:11]: You don't like my pseudospelling?

File: client/src/GamemasterInGame/PlayerListComponents/Selectors/AlignmentSelect.tsx [hunk 3]
```diff
@@ -2,29 +2,38 @@ import { Button, Dialog, Flex } from "@radix-ui/themes";
...
-          size="3"
-          className="capitalize"
-          color={alignmentColorMap[currentAlignment]}
-        >
-          <Flex className="w-full text-center" align="center" justify="center">
-            {currentAlignment}
-          </Flex>
-        </Button>
+        {children || (
+          <Button
```
Comment (Alphy11) [inline client/src/GamemasterInGame/PlayerListComponents/Selectors/AlignmentSelect.tsx:22]: This is generally a bad pattern, instead you just export an AlignmentIndicator or something that goes inside of this.

**Comment by vermentor** [comment]
Too far out of date

**Pull request closed without merging**